File: ChipmeoPOS/src/lib/types/addon.ts
export interface Addon {
	id: number;
	name: string;
	price: number;
	isActive: boolean;
	createdAt?: string;
	updatedAt?: string;
}

export interface AddonCreateDto {
	name: string;
	price: number;
	isActive: boolean;
}

export interface AddonUpdateDto {
	name: string;
	price: number;
	isActive: boolean;
}

File: foodstore-api/FoodstoreApi.Core/Entities/Addon.cs
namespace FoodstoreApi.Core.Entities;

public partial class Addon
{
    public int Id { get; set; }

    public string Name { get; set; } = null!;

    public decimal Price { get; set; }

    public bool? IsActive { get; set; }

    public DateTime? CreatedAt { get; set; }

    public virtual ICollection<MenuItemAddon> MenuItemAddons { get; set; } = new List<MenuItemAddon>();

    public virtual ICollection<OrderItemAddon> OrderItemAddons { get; set; } = new List<OrderItemAddon>();
}




